conf/defaults.ini

```diff
@@ -1110,6 +1110,10 @@ global_file = 1000
 # global limit of correlations
 global_correlations = -1
 
+# Limit of the number of alert rules per rule group.
+# This is not strictly enforced yet, but will be enforced over time.
+alerting_rule_group_rules = 100
```

Comment:
Is there a reason why you didn't name it with something ending on `*_limit`, something like `rules_per_rule_group_limit `?

Reply:
It's inconsistent for Grafana users, Grafana uses the term `quota` instead of `limit` for basically everything.

The current naming is consistent with the existing one:
```
# global limit of alerts
global_alert_rule = -1
```
and also consistent with the other limits in this section.

Reply:
I missed that it was in the quota part of the config, thought we are in the ua part.